lang/c++/test/unittest.cc

```diff
@@ -140,30 +140,30 @@ struct TestSchema
         NodePtr node = schema_.root();
 
         size_t index = 0;
-        bool found = node->nameIndex("mylongxxx", index);
+        bool found = node->nameIndex(std::string("mylongxxx"), index);
```

Comment:
We typically use a convention when the same name from a namespace is used multiple times in `.cc` file: have a `using` declaration at the top and use the plain name.